tasks/noauto_create_cronjobs.yml

```diff
@@ -0,0 +1,71 @@
+---
+- name: Add cron-job for borgmatic (large repo, create and check separate)
+  when:
+    - install_backup is not defined or install_backup
+    - borgmatic_timer is defined and borgmatic_timer == "cron"
+    - borgmatic_large_repo
+  tags:
+    - install_backup
+  block:
+    # I don't like to re-do the installation of packages here, but cron 
+    # should not be one of the automatically installed packages since there
+    # is systemd-timer as an alternative.
+    - name: Install general dependencies (cron)
+      ansible.builtin.package:
+        name: "{{ borg_cron_package }}"
+        state: present
+
+    - name: Add cron job for regular create and prune
```

Comment:
Same as for the Systemd timer. Frequency of repo checks can be adjusted in Borgmatic since ~last year:

https://torsion.org/borgmatic/docs/how-to/deal-with-very-large-backups/#check-frequency